src/Okta.Sdk.IntegrationTests/EventHooksClientShould.cs

```diff
@@ -0,0 +1,382 @@
+            {
+                Uri = "https://www.example.com/eventHooks",
+                Headers = new List<IEventHookChannelConfigHeader>
+                {
+                    new EventHookChannelConfigHeader
+                    {
+                        Key = "X-Test-Header",
+                        Value = "Test header value",
+                    },
+                },
+                AuthScheme = new EventHookChannelConfigAuthScheme
+                {
+                    Type = "HEADER",
+                    Key = "Authorization",
+                    Value = "Test-Api-Key",
+                },
+            },
+        };
+
+        private static readonly EventHookChannel UpdatedTestEventHookChannel = new EventHookChannel
+        {
+            Type = "HTTP",
+            Version = "1.0.0",
+            Config = new EventHookChannelConfig
+            {
+                Uri = "https://www.example.com/eventHooksUpdated",
+                Headers = new List<IEventHookChannelConfigHeader>
+                {
+                    new EventHookChannelConfigHeader
+                    {
+                        Key = "X-Test-Header",
+                        Value = "Test header value updated",
+                    },
+                },
+                AuthScheme = new EventHookChannelConfigAuthScheme
+                {
+                    Type = "HEADER",
+                    Key = "Authorization",
+                    Value = "Test-Api-Key-Updated",
+                },
+            },
+        };
+
+        public EventHooksClientShould()
+        {
+            DeleteAllEventHooks().Wait();
+        }
+
+        [Fact]
+        public async Task CreateEventHook()
+        {
+            var testClient = TestClient.Create();
+            var testEventHookName = $"{SdkPrefix}:{nameof(CreateEventHook)} Test Event Hook Name";
+
+            var createdEventHook = await testClient.EventHooks.CreateEventHookAsync(new EventHook
+            {
+                Name = testEventHookName,
+                Events = new EventSubscriptions
+                {
+                    Type = EventType,
+                    Items = TestEventItems,
+                },
+                Channel = TestEventHookChannel,
+            });
+
+            try
+            {
+                createdEventHook.Id.Should().NotBeNullOrEmpty();
+                createdEventHook.Name.Should().Be(testEventHookName);
+                createdEventHook.Events.Should().NotBeNull();
+                createdEventHook.Events.Items.Should().NotBeNull();
+                createdEventHook.Events.Items.Count.Should().Be(TestEventItems.Length);
+                createdEventHook.Channel.Should().NotBeNull();
+                createdEventHook.Channel.Config.Should().NotBeNull();
+                createdEventHook.Channel.Config.Uri.Should().Be(TestEventHookChannel.Config.Uri);
+            }
+            finally
+            {
+                await DeleteAllEventHooks();
```

Comment:
Nit: Per [MS guideline](https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#avoid-logic-in-tests) we should avoid logical conditions in tests (`foreach` in this case). I recommend removing the specific Event Hook you created on each specific test. That way, if a test fails, we can identify faster what particular entity is failing and analyze the reasons.

Reply:
Removed DeleteAllEventHooks method in favor of individual deletes per test.